devApps/XFormsApp.Droid/XFormsApp.Droid.csproj

```diff
@@ -37,7 +37,7 @@
     <EmbedAssembliesIntoApk>True</EmbedAssembliesIntoApk>
     <BundleAssemblies>False</BundleAssemblies>
     <AndroidCreatePackagePerAbi>False</AndroidCreatePackagePerAbi>
-    <AndroidSupportedAbis>armeabi,armeabi-v7a,x86</AndroidSupportedAbis>
+    <AndroidSupportedAbis>armeabi-v7a,x86</AndroidSupportedAbis>
```

Comment:
Unrelated change: armeabi has been deprecated starting in VS2019. No impact on VS2017.